charts/argo-services/cd-notifications-config.yaml

```diff
@@ -1,33 +1,33 @@
 # NOTE: this file is pulled into templates/argocd-notifications/cm.yaml
 # to avoid needing to nesting conflicting template tags (Helm uses Go, Argo
 # uses Jinja2 which both use the `{{}}` syntax).
-service.webhook.argo_events:
+service.webhook.argo_events: |
   headers:
   - name: "Content-Type"
     value: "application/json"
   url: "http://sync-notification-eventsource-svc.cluster-services.svc.cluster.local:12000"
-service.webhook.slack_webhook:
+service.webhook.slack_webhook: |
   headers:
   - name: "Content-Type"
     value: "application/json"
   url: "$slack_url"
-template.send-argo-events-webhook:
+template.send-argo-events-webhook: |
   webhook:
     argo_events:
       body: |
         {
           "application": "{{.app.metadata.name}}",
           "state": "{{.app.status.operationState.phase}}",
           "commitSha": "{{.app.status.sync.revision}}",
-          "commitUrl": "https://github.com/alphagov/govuk-helm-charts/commits/"{{.app.status.sync.revision}}",
-          "argoUrl": "{{.context.argocdUrl}}/applications/"{{.app.metadata.name}}",
+          "commitUrl": "https://github.com/alphagov/govuk-helm-charts/commits/{{.app.status.sync.revision}}",
+          "argoUrl": "{{.context.argocdUrl}}/applications/{{.app.metadata.name}}",
           "slackChannel": "{{.app.metadata.annotations.slackChannel}}"
         }
       method: "POST"
       path: "/argocd"
-template.send-slack:
-  webhook":
-    slack_webhook":
+template.send-slack: |
+  webhook": |
```

Comment:
What's up with these unbalanced quotes? (similarly below)

Reply:
Oops, sorry about that -- fixed!